tests/queries/0_stateless/02158_groupsortedarray.reference

```diff
@@ -0,0 +1,12 @@
+[0,1,2,3,4]
```

Comment:
Seems like the expected order is different from topK:

```
Mordor :) SELECT groupSortedArrayWeighted(number, number) from numbers(100);

SELECT groupSortedArrayWeighted(number, number)
FROM numbers(100)

Query id: cf18af29-2f70-4400-9aff-fbf04530e8ac

┌─groupSortedArrayWeighted(number, number)─┐
│ [0,1,2,3,4,5,6,7,8,9]                    │
└──────────────────────────────────────────┘

1 rows in set. Elapsed: 0.011 sec. 

Mordor :) SELECT topKWeighted(number, number) from numbers(100);

SELECT topKWeighted(number, number)
FROM numbers(100)

Query id: 2ee96f56-9478-457b-9ecb-71ee0d9a3ce8

┌─topKWeighted(number, number)────┐
│ [99,98,97,96,95,94,93,92,91,90] │
└─────────────────────────────────┘

1 rows in set. Elapsed: 0.009 sec. 
```

I'd expect to be exactly the same order.

Reply:
Now that we realised 'weight' doesn't mean 'orderby' we may change the first to use a more appropiate (orderby/ sort /any).
But I would say it is ok to use this order (ascend).